src/Base/CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ if(MSVC)
         Rpcrt4.lib
         version.lib
     )
+
+    #Universal C runtime introduced in VS 2015 (cl version 19)
```

Comment:
This check fails. Here I have VS 2013 and the output of the variable is "18.0.40629.0" which is already greater than "18"
It looks lie cmake has no "VERSION EQUAL or GREATER" so you have to use "NOT VERSION_LESS" than "19"